Python/ceval.c

```diff
@@ -44,6 +44,32 @@
 #  error "ceval.c must be build with Py_BUILD_CORE define for best performance"
 #endif
 
+#ifndef Py_DEBUG
```

Comment:
if only MSVC is affected, only redefine these macros for MSVC, no?

```suggestion
#if defined(_MSC_VER) && !defined(Py_DEBUG)
```

Reply:
I prefer us running the same code on all platforms if possible, so that it's less likely that someone accidentally breaks the Windows version by want appears to be an innocent change on Linux.

(I have to make an exception for load_relaxed because it is heavily platform-specific.)

Reply:
To be extra safe, you may also check Py_REF_DEBUG, even if Py_REF_DEBUG *should not* be defined if Py_DEBUG is defined.

```suggestion
#if !defined(Py_DEBUG) && !defined(Py_REF_DEBUG)
```

Or maybe object.h should fail with ``#error`` with Py_REF_DEBUG is defined whereas Py_DEBUG is not.